chart/values.yaml

```diff
@@ -93,16 +93,6 @@ thanos:
   enabled: true
   queryFrontend:
     enabled: false
-    extraFlags:
-    - --query-range.split-interval=12h
-    - --query-frontend.log-queries-longer-than=10s
-    - --query-frontend.compress-responses
-    - |-
-      --query-range.response-cache-config="config":
-        "max_size": "500MB"
-        "max_size_items": 0
-        "validity": 0s
-      "type": "in-memory"
```

Comment:
IMO we should keep these `extraFlags` here. 
IMO we should support the scenario when the end-user uses the base values and overrides some parameters like `queryFrontend.enabled=True`, then the end user should end with `extraFlags` we set for `queryFrontend`